homeassistant/components/forecast_solar/const.py

```diff
@@ -26,6 +26,13 @@
         device_class=SensorDeviceClass.ENERGY,
         native_unit_of_measurement=UnitOfEnergy.KILO_WATT_HOUR,
     ),
+    ForecastSolarSensorEntityDescription(
+        key="energy_production_today_remaining",
+        name="Estimated energy production - remaining today",
+        state=lambda estimate: estimate.energy_production_today_remaining / 1000,
```

Comment:
We don't have to preform calculations anymore. Instead, use the provided native value, and set the matching native unit of measurement. If you want to default it to kWh, set the suggested unit of measurement as well.

Reply:
I have changed the unit of measurement to Wh with suggested units kWh. I have changed also all other energy sensors (so I kindly ask Nicolaas for new review).
Since the suggested unit of measurements I think there is no need to change docs.

@frenck  @klaasnicolaas Do You both agree?